data/geany.glade

```diff
@@ -3498,6 +3516,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="tooltip_text" translatable="yes">Display height in rows for the autocompletion list</property>
+                                        <property name="invisible_char">●</property>
```

Comment:
-1, better to add a few noise lines than to manually edit the glade file and push the problem onto the next guy like the problems we had before, IMO. It's just a generated file anyway, not like it's actual code people should be reading :)